dev-requirements.txt

```diff
@@ -5,5 +5,5 @@ boto3>=1.7.28
 #google-cloud==0.33.1
 google-cloud-storage==1.6.0
 -e git+https://git@github.com/uc-cdis/cdislogging.git@0.0.2#egg=cdislogging
--e git+https://git@github.com/uc-cdis/indexclient.git@1.5.6#egg=indexclient
+-e git+https://git@github.com/uc-cdis/indexclient.git@1.5.9#egg=indexclient
```

Comment:
do we need to bump this a a lot?

Reply:
yeah. It is the latest version and more stable than the previous one